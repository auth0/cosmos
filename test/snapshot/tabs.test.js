import React from 'react'
import renderer from 'react-test-renderer'
import { Tabs } from '@auth0/cosmos'
import 'jest-styled-components'

describe('Tabs', () => {
  it('renders with automation attribute', () => {
    const tree = renderer
      .create(
        <Tabs onSelect={() => {}} selected={1}>
          <Tabs.Tab label="Tab 1">This is tab 1</Tabs.Tab>
        </Tabs>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
