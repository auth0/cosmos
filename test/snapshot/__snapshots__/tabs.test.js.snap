// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tabs renders with automation attribute 1`] = `
.c0 {
  border-bottom: 1px solid #D9D9D9;
}

.c0 .c1:last-child {
  margin-right: 0;
}

.c2 {
  display: inline-block;
  padding: 16px 0;
  margin-right: 32px;
  color: #0a84ae;
  cursor: pointer;
  border-bottom: 1px solid transparent;
  margin-bottom: -1px;
}

.c2:hover {
  color: #053b4e;
}

<div
  className=""
  data-cosmos-key="tabs"
>
  <div
    className="c0"
  >
    <a
      className="c1 c2"
      data-cosmos-key="tabs.item"
      onClick={[Function]}
      selected={false}
    >
      Tab 1
    </a>
  </div>
</div>
`;
